lib/src/index_impl.dart

```diff
@@ -181,7 +181,7 @@ class MeiliSearchIndexImpl implements MeiliSearchIndex {
     return await _update(http.putMethod(
       '/indexes/$uid/documents',
       data: documents,
-      queryParameters: <String, dynamic>{
+      queryParameters: {
```

Comment:
since `queryParameters` already defines a type, it can be removed here safely, I probably caught this later on and forgot to change the others as well

an example for a bad case would be:
```dart
final someVar = {'key': true};
```
it should be:
```dart
final someVar = <String,Object?>{'key': true};
```
or 
```dart
final Map<String,Object?> someVar = {'key': true};
```